ntp-proto/src/packet/v5/extension_fields.rs

```diff
@@ -21,6 +22,7 @@ pub enum Type {
 }
 
 impl Type {
+    #[cfg(test)]
```

Comment:
And why is this needed?

Reply:
This function is only used in tests, in normal usage we just use the `fn from_type_id` in the regular `extension_fields.rs`, so I do think we probably need to do some more cleanup here, but I considered that out of scope for this PR (it is big enough already)